extensions/liquibase/runtime/src/main/java/io/quarkus/liquibase/runtime/LiquibaseConfig.java

```diff
@@ -0,0 +1,80 @@
+package io.quarkus.liquibase.runtime;
+
+import static io.quarkus.liquibase.runtime.LiquibaseDataSourceBuildTimeConfig.DEFAULT_CHANGE_LOG;
+import static io.quarkus.liquibase.runtime.LiquibaseDataSourceRuntimeConfig.DEFAULT_LOCK_TABLE;
+import static io.quarkus.liquibase.runtime.LiquibaseDataSourceRuntimeConfig.DEFAULT_LOG_TABLE;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * The liquibase configuration
+ */
+public class LiquibaseConfig {
```

Comment:
If this is a Quarkus config class, the annotations are missing.

Reply:
The QLiquibase contains the LiquibaseConfig object which is merge of the run-time configuration and build configuration. It could be replaced by the run-time and build configuration.